apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/BuiltinAdapters.kt

```diff
@@ -0,0 +1,113 @@
+package com.apollographql.apollo.api
+
+import com.apollographql.apollo.api.internal.json.JsonWriter
+import com.apollographql.apollo.api.internal.json.Utils
+import com.apollographql.apollo.api.internal.json.use
+import okio.Buffer
+
+/**
+ * Builtin CustomScalarTypeAdapter provided for convenience. Encoding is most of the times straightforward. Decoding
+ * can involve coercion. If you need stricter decoding or different logic, define your own adapter
+ */
+object BuiltinScalarTypeAdapters {
+  val STRING_ADAPTER = adapterWithDefaultEncode { jsonElement ->
```

Comment:
The builtin CustomScalarTypeAdapters are now public so that users can re-use them if required.